install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,45 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
```

Comment:
@gaurav-nelson I think instead of deleting the "an", I'd replace with "a". It's still needed.